ios/NativeSigner/Backend/ExportMultipleKeysService.swift

```diff
@@ -26,14 +26,39 @@ final class ExportMultipleKeysService {
         self.callbackQueue = callbackQueue
     }
 
-    func exportMultipleKeys(
-        items: [String]? = nil,
+    func exportMultipleKeySets(
+        seedNames: [String],
+        _ completion: @escaping (Result<AnimatedQRCodeViewModel, ServiceError>) -> Void
+    ) {
+        let selectedItems: [String: ExportedSet] = Dictionary(
+            uniqueKeysWithValues: seedNames
+                .map { ($0, ExportedSet.all) }
+        )
+        export(selectedItems: selectedItems, completion)
+    }
+
+    func exportRootWithDerivedKeys(
+        seedName: String,
+        keys: [MKeyAndNetworkCard],
+        _ completion: @escaping (Result<AnimatedQRCodeViewModel, ServiceError>) -> Void
+    ) {
+        let selectedItems: [String: ExportedSet] = [
+            seedName: .selected(s: keys.map(\.asPathAndNetwork))
+        ]
+        export(selectedItems: selectedItems, completion)
+    }
+
+    private func export(
+        selectedItems: [String: ExportedSet],
```

Comment:
@Dmitry-Borodin could you push update to this PR that would use `[String: ExportedSet]` instead of `List<String>` within `exportKeyInfo` function in `UniffiInteractor.kt`? 🙏🏻 

On "Key Sets" screen, we should be using it as a dictionary where keys are selected seeds and value is enum value: `ExportedSet.all`

Reply:
done